signal/signal.go

```diff
@@ -84,7 +86,7 @@ func executeRunner(c config.Config) error {
 	for processed <= workers {
 		select {
 		case r := <-done:
-			if c.TestFlag {
+			if len(c.TestURL) > 0 {
```

Comment:
nit:
checking string length seems to be a correct way in go, i'd probably prefer something like `if c.TestURL != ""`
it just makes me confident i work with a string and not a slice or map